framer/Material-UI.framerfx/code/Avatar.tsx

```diff
@@ -75,14 +73,14 @@ addPropertyControls(Avatar, {
     type: ControlType.Image,
     title: 'Image File',
     hidden: function hidden(props) {
-      return props.primaryAction && props.primaryAction !== 'avatar';
+      return false;
```

Comment:
Why this change?

Reply:
Right, I wanted to ask: According to the types there's no `primaryAction` so this would always evaluate to `false`. What type should `primaryAction` have?

Reply:
Ah, right. Looks like a bug. We should be applying `avatarImageFile` and `avatarImageUrl` from `scripts/additionalProps.js`  instead of `imageFile` and `imageUrl`. It needs to be changed in `scripts/framerConfig.js`, and the avatar template updated to bind to the correct name in the props destructuring (similar to badge).

Reply:
fixed